docs/user-installation-review.md

```diff
@@ -0,0 +1,211 @@
+* [Azure](https://github.com/openshift/installer/blob/master/data/data/azure/vnet/internal-lb.tf#L7): Standard
+
+### Review the private Load Balancer <a name="loadbalancers"></a>
+
+The basic OpenShift Installations with support of external Load Balancers deploy 3 Load Balancers: public and private for control plane services (Kubernetes API and Machine Config Server), and one public for the ingress.
+
+The DNS or IP address for the private Load Balancer must point to the DNS record `api-int.<cluster>.<domain>`, which will be accessed for internal services.
+
+Reference: [User-provisioned DNS requirements](https://docs.openshift.com/container-platform/4.11/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-dns-user-infra_installing-platform-agnostic)
+
+### Review Health Check configurations <a name="loadbalancers-healthcheck"></a>
+
+The kube-apiserver has a graceful termination engine that requires the Load Balancer health check probe to the HTTP path.
+
+| Service | Protocol | Port | Path | Threshold | Interval | Timeout |
+| -- | -- | -- | -- | -- | -- | -- |
+| Kubernetes API Server | HTTPS* | 6443 | /readyz | 2  | 10 | 10 |
+| Machine Config Server | HTTPS* | 22623 | /healthz | 2  | 10 | 10 |
+| Ingress | TCP | 80 | - | 2  | 10 | 10 |
+| Ingress | TCP | 443 | - | 2  | 10 | 10 |
+
+<!-- > Note/Question: Not sure if we need to keep the HTTP (non-SSL on the doc). In the past, I talked with the KAS team and he had plans to remove that option, but due to the limitation of a few cloud providers, it will not. Some providers that still use this: [Alibaba](https://github.com/openshift/installer/blob/master/data/data/alibabacloud/cluster/vpc/slb.tf#L31), [GCP Public](https://github.com/openshift/installer/blob/master/data/data/gcp/cluster/network/lb-public.tf#L20-L21)
+*It's required to health check support HTTP protocol. If the Load Balancer used does not support SSL, alternatively and not preferably you can use HTTP - but never TCP:
+
+| Service | Protocol | Port | Path | Threshold | Interval | Timeout |
+| -- | -- | -- | -- | -- | -- | -- |
+| Kubernetes API Server | HTTP* | 6080 | /readyz | 2  | 10 | 10 |
+| Machine Config Server | HTTP* | 22624 | /healthz | 2  | 10 | 10 |
+
+-->
+
+
+Reminder for the API Load Balancer Health Check:
+
+*"The load balancer must be configured to take a maximum of 30 seconds from the time the API server turns off the /readyz endpoint to the removal of the API server instance from the pool. Within the time frame after /readyz returns an error or becomes healthy, the endpoint must have been removed or added. Probing every 5 or 10 seconds, with two successful requests to become healthy and three to become unhealthy, are well-tested values."*
+
+
+### Review Hairpin Traffic <a name="loadbalancers-hairpin"></a>
+
+If the Load Balancer does not allow hairpin traffic to the node, when a backend is load-balanced to itself and the traffic is dropped, you need to provide a solution.
```

Comment:
Can this be rewritten to:

Hairpin traffic is when a backend node's traffic is load-balanced to itself. If this type of network traffic is dropped because your load balancer does not allow hairpin traffic, you need to provide a solution.

Reply:
Cool, thanks